pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -1383,13 +1382,15 @@ func initializeDirs() {
 // Run is the entry point for tpmmgr, from zedbox
 //
 //nolint:funlen,gocognit,gocyclo
-func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, arguments []string) int {
+func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, arguments []string, baseDir string) int {
 	logger = loggerArg
 	log = logArg
 
 	// Context to pass around
 	ctx := tpmMgrContext{}
 	agentbase.Init(&ctx, logger, log, agentName,
+		agentbase.WithPidFile(),
```

Comment:
No, because when runInline below there shouldn't be a pidfile created.

Reply:
But it existed just a few lines below at 1403. Why would this break it?

Reply:
Oh I see. I missed the inline conditional after. Got it. This helps greatly. Thank you. I will check others as well.